src/openms/include/OpenMS/DATASTRUCTURES/ListUtils.h

```diff
@@ -134,6 +134,22 @@ namespace OpenMS
     template <typename T>
     static std::vector<T> create(const std::vector<String>& s);
 
+
+    /**
+  @brief Converts a vector of T's to a vector of Strings.
+
+  @param s The vector of T's that should be converted.
+  @return A vector containing the elements of input vector converted into Strings.
+*/
+    template <typename T>
+    static std::vector<String> toStringList(const std::vector<T>& s)
```

Comment:
could we also make the container a template so it can be used with lists and sets?

Reply:
maybe too much trouble but boost has some traits for that

Reply:
My preferred solution would be a StringList class which has appropriate members. Simply because it's easier to find functionality than using free functions.
But that was too much trouble for now. But might make for a nice hacktoberfest project.

I'd like to avoid boost for such simple things because of unneeded dependency and the header bloat which boost usually entails. But that's just me.